packages/afterpay-integration/e2e/Afterpay.spec.ts

```diff
@@ -1,16 +1,18 @@
-import { test, PaymentStepAsGuestPreset, UseAUDPreset } from '@bigcommerce/checkout/payment-integration-test-framework';
+import { expect, test, PaymentStepAsGuestPreset, UseAUDPreset } from '@bigcommerce/checkout/payment-integration-test-framework';
 
 test.describe('Sample Test Group', () => {
-    test('Can see Afterpay AU on the payment step', async ({assertions, checkout}) => {
+    test('Can see Afterpay AU on the payment step', async ({assertions, checkout, page}) => {
         // Testing environment setup
         await checkout.use(new UseAUDPreset('AUD'));
         await checkout.use(new PaymentStepAsGuestPreset());
         await checkout.start('sample Afterpay AUD');
 
         // Playwright actions
         await checkout.goto();
+        await checkout.selectPaymentMethod('pay_by_installment');
 
         // Assertions
         await assertions.shouldSeePaymentStep();
+        await expect(await page.locator('#radio-afterpay-pay_by_installment').isChecked()).toBeTruthy()
```

Comment:
Checking `checkout.selectPaymentMethod('pay_by_installment')` is actually working.